pkg/exchange/coinbase/api/v1/order.go

```diff
@@ -0,0 +1,47 @@
+package coinbase
+
+import (
+	"context"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+func (client *RestAPIClient) GetOrders(ctx context.Context, symbol string, status []string, limit int,
+	sortedBy *string, sorting *string, before *time.Time) (OrderSnapshot, error) {
+	req := GetOrdersRequest{
+		client:    client,
+		productID: &symbol,
+		status:    status,
+		limit:     limit,
+		sortedBy:  sortedBy,
+		sorting:   sorting,
+		before:    before,
+	}
```

Comment:
try not to pass these values, you can use the accessor methods to assign values from the exchange layer. .e.g.

    client.NewGetOrdersRequest().Sorting("...").Limit(10).Before(...).Do(ctx)

Reply:
Done.